cognito/variables.tf

```diff
@@ -41,4 +41,55 @@ variable "namespace" {
 }
 
 variable "owner" {
-}
\ No newline at end of file
+}
+
+# admin_create_user_config
+variable "admin_create_user_config" {
+  description = "The configuration for AdminCreateUser requests"
+  type        = map
+  default     = {}
+}
+
+variable "admin_create_user_config_allow_admin_create_user_only" {
+  description = "Set to True if only the administrator is allowed to create user profiles. Set to False if users can sign themselves up via an app"
+  type        = bool
+  default     = false
+}
+
+variable "admin_create_user_config_unused_account_validity_days" {
+  description = "The user account expiration limit, in days, after which the account is no longer usable"
+  type        = number
+  default     = 0
+}
+
+variable "admin_create_user_config_email_message" {
+  description = "The message template for email messages. Must contain `{username}` and `{####}` placeholders, for username and temporary password, respectively"
+  type        = string
+  default     = "{username}, your verification code is `{####}`"
+}
+
+
+variable "admin_create_user_config_email_subject" {
+  description = "The subject line for email messages"
+  type        = string
+  default     = "Your verification code"
```

Comment:
can this default set to null?

Reply:
I haven't tried but I would surprised since the AWS cognito interface itself will not create it as null.